examples/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "dependencies": {
-        "@angular/compiler-cli": "^18",
+        "@angular/compiler-cli": "18.1.0-next.0",
```

Comment:
Why change this and only this one?

Reply:
Ah, it's required since Angular 18.0 has a TS upper bound of <5.5.0
I could bump the other two packages but only the compiler-cli needs the fix.
Temporary until Angular 18.1